### DISCUSSION TITLE
I want to translate the spanish lanuage text to speech/audio mp3 file but what is the model value i need to provide whil calling curl url ?

### DISCUSSION THREAD

**Discussion by ganeshvidyayug** [deepgram/community#776] [General help] [answered]
curl -X POST -H "Authorization: Token 76b8b6b9935790a635d69be1825dde97bbf5fa5d" -H "Content-Type: text/plain" -d "Deepgram is great for real-time conversations… and also, you can build apps for things like customer support, logistics, and more. What do you think of the voices?" "https://api.deepgram.com/v1/speak?model=aura-asteria-es" -o audio.mp3

here model=aura-asteria-es is not working to translate spanish text to spanish audio, please help me out,  am getting below error
{"err_code":"INSUFFICIENT_PERMISSIONS","err_msg":"Project does not have access to the requested model.","request_id":"f6f72762-a8a6-4fbc-97f6-e4c23e9dfbcf"}, but what is the value I need to give for model to translate spanish text to spanish audio

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by ganeshvidyayug** [reply]
curl -X POST -H "Authorization: Token 76b8b6b9935790a635d69be1825dde97bbf5fa5d" -H "Content-Type: text/plain" -d "El título de la infografía es difusión de la innovación y el proceso de adopción del teléfono. Al inicio del proceso de adopción sólo el 2,5% son innovadores y dicen que es fantástico hablar a distancia. Luego, los primeros usuarios con el 13,5% de los usuarios. Dicen que estoy emocionado de probar esto. Sólo desearía que hubiera más personas a quienes llamar. En este punto, los usuarios de Summit optan por dejar de usar la nueva innovación, otros continúan usando el teléfono y se les considera la primera mayoría de usuarios. Son el 34% de los usuarios y dicen que estamos adoptando esto en todo nuestro negocio. Luego viene la última mayoría de usuarios, que son el 34% de los usuarios. Dicen que parece que todo el mundo tiene un teléfono, deberíamos conseguir uno. El último grupo son los rezagados y representan el 16% de los usuarios. Dicen qué hay de malo en escribir una carta." "https://api.deepgram.com/v1/speak?model=nova-2-general" -o audioen3.mp3                                                                                                                                        ---> here am sharing the curl request

**Reply by davidvonthenen** [reply]
@ganeshvidyayug  FYI, I would revoke your API Key since it now is exposed.

**Comment by SandraRodgers** [comment, marked as answer]
Hello @ganeshvidyayug ,

You are attempting to use our Text-to-Speech API with Spanish, but we only offer English right now. Spanish is coming soon. For the time being, it won't work to add `es` to the model like `aura-asteria-es`. You can see the available models here: https://developers.deepgram.com/docs/tts-models

You could also fill out [this survey](https://deepgram.typeform.com/aura-languages?typeform-source=developers.deepgram.com) to let the team know what languages you would like us to offer.
